legacy/public/js/airtime/tracktype/tracktype.js

```diff
@@ -156,6 +150,7 @@ function initTracktypeData() {
     description: "",
     visibility: "1",
     id: "",
+    analyze_cue_points: true,
```

Comment:
This seem unrelated ?

Reply:
Err yes. This is part of a bigger thing i am working on. However this field is not initialized in the code as-is. Default for this is 'true' in the schema, so i added it.

Reply:
Please keep this is the "bigger thing" you are working on.

Reply:
Removed, sorry.